### PR TITLE
chore: add setting for spfx version

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
image

### CODE REVIEW

File: packages/vscode-extension/package.json [hunk 1]
```diff
@@ -982,6 +982,12 @@
               "yo": true,
               "sharepointYeomanGenerator": true
             }
+          },
+          "fx-extension.SPFxVersion":{
+            "type": "string",
+            "description": "Select used SPFx version. (Requires reload of VS Code)",
+            "enum": ["1.15.0", "1.16.0-beta.1"],
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/package.json:989]: The exact version for beta will be updated later.

**Comment by HuihuiWu-Microsoft** [comment]
@xu-hong Please help review the string.

File: packages/vscode-extension/package.json [hunk 2]
```diff
@@ -982,6 +982,12 @@
               "yo": true,
               "sharepointYeomanGenerator": true
             }
+          },
+          "fx-extension.SPFxVersion":{
+            "type": "string",
+            "description": "Select used SPFx version. (Requires reload of VS Code)",
```
Comment (xu-hong) [inline packages/vscode-extension/package.json:988]: Select what version of SPFx to use. (Requires reload of VS Code)

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.json]: Updated

**Review by tecton — APPROVED** [review]

**Pull request merged**